perf/benchmark/setup_test.sh

```diff
@@ -73,4 +73,12 @@ for ((i=1; i<=$#; i++)); do
 done
 
 kubectl create ns "${NAMESPACE}" || true
+
+kubectl label namespace "${NAMESPACE}" istio-injection=enabled --overwrite || true
```

Comment:
Why is istio inject unconditional ?
It should also test for the variable.

Reply:
yes, you are right!